simba_ml/prediction/time_series/data_loader/transfer_learning_data_loader.py

```diff
@@ -86,8 +87,12 @@ def X_test(self) -> npt.NDArray[np.float64]:
         """
         if self.__X_test is None:
             self.prepare_data()
-            return self.X_test
-        return self.__X_test
+        if self.config.export_path is not None:
+            if not os.path.exists(os.path.join(os.getcwd(), self.config.export_path)):
+                os.mkdir(os.path.join(os.getcwd(), self.config.export_path))
+            assert self.__X_test is not None
+            export.export_input_batches(self.__X_test, self.config)
+        return self.X_test if self.__X_test is None else self.__X_test
```

Comment:
Isn't __X_test always set?

Reply:
I included this to satisfy mypy. He is also concerned about the return value being None. Do you have another workaround for this?

Reply:
I see, mypy is right here.